eks_cluster.tf

```diff
@@ -1,5 +1,5 @@
 ###########################################
-###...Creation Kubernetes Cluster
+### Kubernetes Cluster. Step 2          ###
 ###########################################
 
 module "kubcluster" {
```

Comment:
```suggestion
module "kubecluster" {
```
shortcut for Kubernete sis kube, not kub

Reply:
Fixed in next PR.